app/api.py

```diff
@@ -38,3 +44,194 @@ def main_page(request: Request):
 @app.get("/robots.txt", response_class=PlainTextResponse)
 def robots_txt():
     return """User-agent: *\nDisallow: /"""
+
+
+# CRUD Flags
+
+
+class TicketCreate(BaseModel):
+    barcode: str = Field(..., description="Barcode of the product")
+    type: str = Field(..., description="Type of the issue")
+    url: str = Field(..., description="URL of the product")
+    status: str = Field(..., description="Status of the ticket")
+    image_id: str = Field(..., description="Image ID of the product")
+    flavour: Flavor = Field(..., description="Flavour of the product")
+    created_at: datetime = Field(default_factory=datetime.utcnow)
+
+
+class Ticket(TicketCreate):
+    id: int = Field(..., description="ID of the ticket")
+
+
+class FlagCreate(BaseModel):
+    barcode: str = Field(..., description="Barcode of the product")
+    type: str = Field(..., description="Type of the issue")
+    url: str = Field(..., description="URL of the product")
+    user_id: str = Field(..., description="User ID of the flagger")
```

Comment:
We can impersonate the user if we send any `user_id`, we should authenticate the user. But we will wait for the new auth system to do this, it's fine like this for now.